### PR TITLE
spooling animation fixes

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: operators.py [hunk 1]
```diff
@@ -278,56 +278,56 @@ def execute(self, context):
...
-                {'INFO'}, 'RenderMan External Rendering rendering ribs ' + str(rib_names))
-            try:
-                process = subprocess.Popen(cmd, cwd=cdir, stdout=subprocess.PIPE,
-                                           stderr=subprocess.PIPE, env=environ)
-            except:
-                self.report({'ERROR'}, 'Error launching prman')
-
-            self.report(
-                {'INFO'}, 'RenderMan External Rendering done rendering.')
+#        if rm.external_action == 'render':
```
Comment (bsavery) [inline operators.py]: Are these supposed to be commented out?  Or is this part of the taking the "render option" away?  If so I'd rather do that as part of a point release.

File: ui.py [hunk 2]
```diff
@@ -305,6 +306,8 @@ def draw(self, context):
         col.enabled = rm.external_action != 'render'
```
Comment (bsavery) [inline ui.py]: Also wouldn;t need this...

File: properties.py [hunk 3]
```diff
@@ -826,11 +826,10 @@ def reset_ll_object_index(self, context):
     external_action = EnumProperty(
         name="Action",
         description="Action for rendering externally.",
-        items=[('render', 'Local render', 'Render to the Display Driver choosen'),
-               ('ribgen', 'Generate RIB only',
+        items=[('ribgen', 'Generate RIB only',
                 'Only Generate RIB and job file (no render)'),
                ('spool', 'Spool Job', 'Spool Job to queuing system')],
-        default='render')
+        default='spool')
```
Comment (bsavery) [inline properties.py]: I agree spool should be default

File: operators.py (hunk 1 above)
Comment (jdent02) [inline operators.py]: Yes, they were supposed to be as part of removing the local render option.  That's cool if you want to save it for a point release.

File: ui.py (hunk 2 above)
Comment (jdent02) [inline ui.py]: True

File: operators.py (hunk 1 above)
Comment (bsavery) [inline operators.py]: Actually you know what.  Let's take it out now.

Comment (jdent02) [inline operators.py]: Delete or leave commented out?

**Comment by bsavery** [comment]
So on the bigger issue of animating geometry archives always going to the same frame # dir.  Is this fixed with that?  I would think the animated rib path on the rpass would have to update?

File: operators.py (hunk 1 above)
Comment (bsavery) [inline operators.py]: Delete

**Comment by bsavery** [comment]
Oh nevermind.  Update_frame_num

**Comment by jdent02** [comment]
Yep.  Tried it out on a cloth deoformation and a particle animation and both worked like a charm

**Comment by bsavery** [comment]
Oh also just make the changes in devel, easier to move the change after.

**Comment by jdent02** [comment]
So you don't want the spooling fixes in master?

**Comment by jdent02** [comment]
Which devel?  The pre 21 one is really out of date now and I though it was depreciated since I assumed the next release was going to integrate 21.

**Comment by bsavery** [comment]
Sorry I meant make the changes to master

**Comment by jdent02** [comment]
Okay.  I'll close this and open a pull for master.

**Pull request closed without merging**